src-tauri/src/northstar/mod.rs

```diff
@@ -145,15 +145,11 @@ pub fn launch_northstar_steam(
                 let titanfall2_steamid: u32 = TITANFALL2_STEAM_ID.parse().unwrap();
                 match steamdir.compat_tool(&titanfall2_steamid) {
                     Some(compat) => {
-                        if !compat
-                            .name
-                            .clone()
-                            .unwrap()
-                            .to_ascii_lowercase()
-                            .contains("northstarproton")
+                        if !VALID_NORTHSTAR_PROTON_BUILDS
+                            .contains(&compat.clone().name.unwrap().as_str())
```

Comment:
@Jan200101 is changing the check to also include casing an issue?

Or to say it differently, was there a reason to ignore case sensitivity in previous version? ^^

Reply:
You'd have to ask @Jan200101, but I think it may have just been there for convenience. During development I had it log the strings of the various proton builds (including northstarproton) and they all matched the name of the folders that they were in, whether that was a coincidence or not, I don't know. But if you follow the standard installation instructions for these versions, it should match up fine without the casing check. If you want to be extra safe, it can be added back in.

Reply:
it don't matter, I just kinda did it to give it a little room for error

Reply:
Sounds like the check is good as it is, marking as resolved.